### PR TITLE
feat: Add session trace metadata params

### PR DESCRIPTION
**Pull request by metal-messiah**
Add first timestamp and node count information to session trace payloads for optimization in UI experiences
---

### Overview
This PR adds the `fst` and `n` query params to the STN payload, which will be digested as `firstTimestamp` and `nodeCount` downstream in the consumer.  These attributes are intended to be helper properties for the UI and API constructing Session Trace visualizations without needing to read and break apart the "blobified" STN payload.

### Related Issue(s)
https://issues.newrelic.com/browse/NR-154847
Sister PR in consumer -- https://source.datanerd.us/browser/bam-consumer/pull/1747

### Testing
New tests have been added to validate this behavior

### CODE REVIEW

File: src/features/session_trace/aggregate/index.js
Comment (cwli24) [inline src/features/session_trace/aggregate/index.js:448]: btw you can also start with `0` and not have to check acc, I think?

Comment (metal-messiah) [reply src/features/session_trace/aggregate/index.js:448]: then nothing would ever be less than 0

File: src/features/session_trace/aggregate/index.js
Comment (metal-messiah) [reply src/features/session_trace/aggregate/index.js:470]: We want to get away from shoving `customer-defined` data into the query yes.  Theres no safety risk of doing it ourselves.

**Review by cwli24 — APPROVED** [review]
Verified the new queries are reporting and values are reasonable.
Should note this actually adds `st`, `fts`, and `n` to `/resource` endpoint URL.

File: src/features/session_trace/aggregate/index.js
Comment (cwli24) [reply src/features/session_trace/aggregate/index.js:448]: oops I meant `Infinity` heh

**Comment by metal-messiah** [comment]
Wait for approval on BST consumer PR before merging this

**Review by patrickhousley — APPROVED** [review]

**Pull request merged**
